ingest_missing.py

```diff
@@ -0,0 +1,155 @@
+"""
+Usage: ingest_missing.py <event_url> <particle_url> <server>
+
+This script will grab dates missing from data ingestion and then it will use the playback command to ingest them.
+Provide the event_url and particle_url. 
+
+Example: python ingest_missing.py qpid://guest/guest@uframe-test?queue=Ingest.instrument_events qpid://guest/guest@uframe-test?queue=Ingest.instrument_particles uframe-test
+
+Arguments:
+    event_url    
+    particle_url
+    server          This is where the reference designator information is stored.
+
+Options:
+    -h --help
+    -v       verbose mode
+    -q       quiet mode
+"""
+
+from docopt import docopt
+import sys
+import time
+import requests
+import re
+import glob
+import list_missing_dates
+from subprocess import call
+from datetime import datetime, timedelta
+
+g_cabled_drivers_raw = []
```

Comment:
Instead of using a global, create a class instead and store as a member variable.